### ISSUE TITLE
signal_canonical_name: Does not check g_signal_stop_emission_by_name

### ISSUE THREAD

**Issue by A6GibKm** [bilelmoussaoui/gobject-lint#113]
The following:

```
  g_signal_stop_emission_by_name (editable, "insert_text");
```

does not trigger the rule.
